python-3.13.yaml

```diff
@@ -0,0 +1,263 @@
+package:
+  name: python-3.13
+  version: 3.13.0_rc2
+  epoch: 0
+  description: "the Python programming language"
+  copyright:
+    - license: PSF-2.0
+  resources:
+    cpu: 8
+    memory: 8Gi
+  dependencies:
+    provider-priority: 0
+    provides:
+      - python3=${{package.full-version}}
+      - python-3=${{package.full-version}}
```

Comment:
```suggestion
```

I don't think we want this yet, until it's meant to become the latest version.

Reply:
thank you. you're right.

Reply:
Or you also rebuild all previous pythons and add a provider priority on them.

Or you make these python3 provides to be =0 rather than `=${{package.full-version}}`

Reply:
i went with python3=0 and that seems to work.